tests/acceptance/tests/Visual/Administration/ProductListLayout.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, setViewport, assertScreenshot } from '@fixtures/AcceptanceTest';
+
+test('Visual: Shopping experiences pages', { tag: '@Visual' }, async ({
+    ShopAdmin,
+    AdminSettingsListing,
+    AdminLayoutListing,
+    AdminLayoutCreate,
+    AdminListingPageLayoutDetail,
+    AdminApiContext,
+}) => {
+    let createdLayoutId: string;
```

Comment:
Why is this not a constant? I can't see that the variable gets changed anywhere.